src/main/java/core/basesyntax/strategy/OperationHandlerStrategy.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.impl.OperationHandlerBalanceImpl;
+import core.basesyntax.strategy.impl.OperationHandlerPurchaseImpl;
+import core.basesyntax.strategy.impl.OperationHandlerReturnImpl;
+import core.basesyntax.strategy.impl.OperationHandlerSupplyImpl;
+import java.util.HashMap;
+import java.util.Map;
+
+public class OperationHandlerStrategy {
+    private static final Map<FruitTransaction.Operation,
+            OperationHandler> STRATEGY_FRUITS_TRANSACTION_OPERATIONS_MAP;
+    private static final Storage STORAGE = new Storage();
```

Comment:
inject in with constructor, it should not be static
```suggestion
    private final Map<FruitTransaction.Operation,
            OperationHandler> handlersMap;
```